README.markdown

```diff
@@ -152,6 +152,14 @@ Sets the instance size to be launched to the requested size. Current valid sizes
 
 Unlike the normal EC2 default, profab has a default size of `t1.micro`.
 
+## volume ##
```

Comment:
This seems like a bad name for something that appears to change the root volume size. Something longer and more descriptive is probably needed: `root_volume_size` maybe